server/commands.lua

```diff
@@ -141,8 +141,10 @@ end, 'user')
 QBCore.Commands.Add('setjob', 'Set A Players Job (Admin Only)', {{name = 'id', help = 'Player ID'}, {name = 'job', help = 'Job name'},{name = 'grade', help = 'Grade'}}, true, function(source, args)
     local src = source
     local Player = QBCore.Functions.GetPlayer(tonumber(args[1]))
+	local PlayerJob = QBCore.Functions.GetPlayer(source).PlayerData.job
```

Comment:
The indentation level doesn't match. Apart from that, you are getting the PlayerJob from the person who issued the command, not the target?